README.md

```diff
@@ -4,26 +4,40 @@
 
 ## Testing
 
-`crystal spec` to run tests
+### Without Docker
 
-## Compiling
-
-`crystal build ./src/rest-api.cr`
+- `crystal spec` to run tests
 
-## Dependencies
+**Dependencies**
```

Comment:
what's the purpose of this change?

Reply:
**Dependencies** required for testing with or without Docker is different, so instead of putting it in h2 (##), I put it as bold, underneath either ### Without ### or ### With Docker ###